mirage-support/factories/show.js

```diff
@@ -0,0 +1,33 @@
+import { Factory, faker } from 'ember-cli-mirage';
+import { dasherize } from 'ember-string';
+
+const slugify = str => dasherize(str.replace(/[^\w\s]/gi, '-'));
+
+function generateProducingOrg() {
+  return {
+    url: faker.internet.url(),
+    name: faker.company.companyName(),
+    logo: {}
+  };
+}
+
+export default Factory.extend({
+  slug: () => slugify(faker.name.findName()),
+  about: {
+    body: '<h1>About</h1>'
+  },
+  title() {
+    return `${faker.name.findName()} Show`;
+  },
+  producingOrganizations() {
+    let org = [generateProducingOrg()];
+
+    if (Math.random() > 0.5) {
+      org.push(generateProducingOrg());
+    }
+
+    return org;
+  },
+  tease: faker.lorem.sentences,
```

Comment:
Because mirage factories automatically pass the id of the created model instance to functions on their properties, the tease for the first show created here is zero sentences long (`faker.lorem.sentences(0)`), and the tease for the 101st show is one hundred sentences long (`faker.lorem.sentences(100)`).

If we don't want this effect, (especially the zero sentence teases), using `() => faker.lorem.sentences()` will work better.

Reply:
oh wow what a great catch!